state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1008,7 +1018,7 @@ pub mod pallet {
 
 		/// Schedule the swap, assuming all checks already passed. Return swap_id along
 		/// with the block at which the swap is scheduled to be executed.
-		fn schedule_swap_internal(
+		pub fn schedule_swap_internal(
```

Comment:
It is not very "internal" if you make it public 😄 Also, looks like the `pub` is not actually needed?

Reply:
I think it can be inlined into the trait function